### DISCUSSION TITLE
Hay link para subir practicas

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#370] [Q&A]
Buena tarde saben si hay link para subir la practica de css

**Comment by Ingrzs** [comment]
No aun no lo publican.

**Comment by sergiolopezloya** [comment]
Yo ya casi tengo listo mi pokedex y aún no está para entregar la de css, jajajaja, a ver si ya estos días. Me parece raro porque publicó hoy el forms para mandar el de la pastelería (con fecha límite de mañana 13), será que es el mismo forms de entrega para el de las vacunas?

**Reply by JuanSalvador73** [reply]
Las preguntas y campos de llenado de las formas son diferentes según lo que pide. Y solo se puede llenar una vez. Creo que en cuanto publiquen el nuevo veremos las nuevas preguntas.

**Comment by GloriaCoralCerecedo** [comment]
El MC comento que se entrega hoy pero que dara más chance
